src/pages/CertificateViewer.js

```diff
@@ -98,7 +98,7 @@ export default function CertificateViewer() {
       alert('DONE')
     }
 
-    // If Emma then post co2 if it hasn't got co2, as in, if no embedded co2 from fetch
+    // If Emma then post co2 if it hasn't got co2 (no embedded co2 from fetch) then start fetch loop
     curPersona === 'emma' &&
       fetchLatestCert().then((c) => {
```

Comment:
why do we need latest cert? I know this might not be part of this PR, so just a question

Reply:
Because I need to information from that GET fetch to make the decision to carry on posting the Co2 or not because it the cert already has Co2 there is no point trying to post it on the chain again because it wouldn't work .. hope that makes sense ..